README.rst

```diff
@@ -20,6 +20,10 @@ CKAN: The Open Source Data Portal Software
     :target: https://coveralls.io/github/ckan/ckan?branch=master
     :alt: Coverage Status
 
+.. image:: https://requires.io/github/ckan/ckan/requirements.svg?branch=master
```

Comment:
Maybe when we get rid of Pylons we can have a badge to brag about up to date requirements, until then it will always show "outdated" which doesn't look too good